alpha/apps/kaltura/modules/extwidget/templates/previewSuccess.php

```diff
@@ -153,6 +153,11 @@ function getParameterByName(name, url) {
             playerConfig = mergeDeep(playerConfig,parsedConfig);
         }
         catch(ee){}
+    }
+    if (typeof data.flashVars !== 'undefined') {
```

Comment:
don't repeat the check, do it once and save the KS and then use it in the dynamic or iframe embeds